libcob/common.c

```diff
@@ -7723,7 +7744,9 @@ translate_boolean_to_int (const char *ptr)
 	if (*(ptr + 1) == 0 && isdigit ((unsigned char)*ptr)) {
 		return atoi (ptr);		/* 0 or 1 */
 	} else
-	if (strcasecmp (ptr, "not set") == 0) {
+	/* pre-translated boolean "never" - not set" */
+	if (strcmp (ptr, "!") == 0) {
+	//if (strcasecmp (ptr, "not set") == 0) {
```

Comment:
that `//` should be dropped - please check if we reasonably want to keep the line and either change comment style or drop